esmvalcore/cmor/_fixes/cmip6/mpi_esm1_2_hr.py

```diff
@@ -0,0 +1,152 @@
+"""Fixes for MPI-ESM1-2-HR model."""
+
+from ..fix import Fix
+from ..shared import add_scalar_height_coord
+
+
+class Tas(Fix):
+    """Fixes for tas."""
+
+    def fix_metadata(self, cubes):
+        """
+        Fix metadata.
+
+        Adds missing height2m coordinate.
+
+        Parameters
+        ----------
+        cube: iris.cube.CubeList
+
+        Returns
+        -------
+        iris.cube.CubeList
+
+        """
+        for cube in cubes:
+            add_scalar_height_coord(cube)
+
+        return cubes
+
+
+class Ta(Fix):
+    """Fixes for ta."""
+
+    def fix_metadata(self, cubes):
+        """
+        Fix metadata.
+
+        Corrects plev coordinate var_name.
+
+        Parameters
+        ----------
+        cube: iris.cube.CubeList
+
+        Returns
+        -------
+        iris.cube.CubeList
+
+        """
+        for cube in cubes:
+            plev = cube.coord('air_pressure')
+            plev.var_name = 'plev'
+
+        return cubes
+
+
+class Va(Fix):
```

Comment:
Maybe you could just replace this by 
```
class Va(Ta):
    pass
```

Also, the very lengthy numpy style docstrings are not really needed, since this is not part of the public api and it really reduces readability in this case.

Reply:
Since they both need the same ones and are the same model but in different resolutions, for the XR, can the fixes from the HR be imported? I mean like it's done in the cesm2 and cesm2-wacc fixes. Or is it better not to mix imports between models?

Reply:
>  can the fixes from the HR be imported?

Yes, I think that should be fine.